docusaurus/docs/e2e-test-a-plugin/migrate-from-grafana-e2e.md

```diff
@@ -0,0 +1,172 @@
+
+### Step 2: Installing `@grafana/plugin-e2e`
+
+Open the terminal and run the following command in your plugin's project directory:
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2InstallNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EInstallYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EInstallPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+### Step 3: Configure Playwright
+
+Open the Playwright config file that was generated when Playwright was installed.
+
+1. Uncomment `baseUrl` and change it to `'http://localhost:3000'`.
+
+```ts title="playwright.config.ts"
+  baseURL: 'http://localhost:3000',
+```
+
+2. Playwright uses [projects](https://playwright.dev/docs/test-projects) to logically group tests that have the same configuration. We're going to add two projects:
+
+   1. `auth` is a setup project that will log in to Grafana and store the authenticated state on disk.
+   2. `run-tests` runs all the tests in your browser of choice. By adding a dependency to the `auth` project we ensure that login only happens once, and all tests in the `run-tests` project will start as already authenticated.
+
+   Your Playwright config should have the following project configuration:
+
+```ts title="playwright.config.ts"
+import { dirname } from 'path';
+import { defineConfig, devices } from '@playwright/test';
+
+const pluginE2eAuth = `${dirname(require.resolve('@grafana/plugin-e2e'))}/auth`;
+
+export default defineConfig({
+    ...
+    projects: [
+    {
+      name: 'auth',
+      testDir: pluginE2eAuth,
+      testMatch: [/.*\.js/],
+    },
+    {
+      name: 'run-tests',
+      use: {
+        ...devices['Desktop Chrome'],
+        // @grafana/plugin-e2e writes the auth state to this file,
+        // the path should not be modified
+        storageState: 'playwright/.auth/admin.json',
+      },
+      dependencies: ['auth'],
+    }
+  ],
+});
+```
+
+The authenticated state is stored on disk with the following file name pattern: `<plugin-root>/playwright/.auth/<username>.json`.
+
+To prevent these files from being version controlled, you can add the following line to your `.gitignore` file:
+
+```shell title=".gitignore"
+/playwright/.auth/
+```
+
+The `@grafana/plugin-e2e` package also exports a type named `PluginOptions` that you can use to extend the Playwright configuration with the `@grafana/plugin-e2e` specific options.
+
+```ts title="playwright.config.ts"
+import { defineConfig } from '@playwright/test';
+import type { PluginOptions } from '@grafana/plugin-e2e';
+
+export default defineConfig<PluginOptions>({
+    ...
+});
+```
```

Comment:
nit: I would combine this with the above playwright.config.ts snippet so it's a copy and paste once rather than copy and paste then manually update with this typing.